README.md

```diff
@@ -1,5 +1,13 @@
 # Suomi.fi Design System
 
-Documentation website for [suomifi-ui-components](https://github.com/vrk-kpa/suomifi-ui-components)
+Documentation website for the [Suomi.fi Design System](https://vrk-kpa.github.io/suomifi-design-system/fi/)
 
-## [Development](DEVELOPMENT.md)
+Built using [suomifi-ui-components](https://github.com/vrk-kpa/suomifi-ui-components)
+
+You can see the current development build at (https://vrk-kpa.github.io/suomifi-design-system/fi/)
+
+The site presents the elements, features and possibilities of the Suomi.fi Design System. For more information check out the site itself from the above link.
```

Comment:
Maybe add something about the contents of the page as well, instead of mentioning possibilities we could write something about the site like overall description of the designs system, styles, components and how to use the DS/Styles/Components.

Reply:
I'd leave most of that to the site itself, but it might be worth it to give a bit more information on what kind of info the reader can find by visiting the site. Good point.